src/veritysetup/veritysetup.c

```diff
@@ -31,7 +31,7 @@ static int help(void) {
         if (r < 0)
                 return log_oom();
 
-        printf("%s attach VOLUME DATADEVICE HASHDEVICE ROOTHASH [ROOTHASHSIG]\n"
+        printf("%s attach VOLUME DATADEVICE HASHDEVICE ROOTHASH [ROOTHASHSIG] [OPTIONS]\n"
```

Comment:
I think the fact you can pass options and omit roothashsig by using "-" needs to be documented here, as it's not obvious

Reply:
Indeed. I would like to know if this would be possible to bypass `ROOTHASHSIG` using `-`. `-` may mean read from stdin; that is not the case here. Personnaly I find this uggly to have two optional parameters, it is the easiest way to implement generator side :) I am not against implementing options in the form `--long-option-name`, it will requires more code in the generator and I am still discovering how things work.

Reply:
Given there are multiple verity options supported, continuing this way essentially means we can't extend it anymore in the future, should the need manifest. So I'd be very much in favour of adding --log-option-name support. Would you be up for it?

Reply:
Sure I will do it. I have planned to implement most of the options offered by setupverity.